### ISSUE TITLE
Studio crashes when viewing history

### ISSUE THREAD

**Issue by jorngeorg** [sanity-io/sanity#1454]
Reported by community user: https://sanity-io-land.slack.com/archives/C9Z7RC3V1/p1566302304196300

> when I attempt to view the history of any long-lived document in my current project, the studio crashes. This seems to be because it cant find a user to relate a change to. 

**Screenshots**
![image (1)](https://user-images.githubusercontent.com/968695/63347353-37f6c100-c357-11e9-8f87-8c93730abf7a.png)
![image (2)](https://user-images.githubusercontent.com/968695/63347397-4ba22780-c357-11e9-9454-18db7471357a.png)

**Which versions of Sanity are you using?**

@sanity/cli                 0.141.6 (latest: 0.142.0)
@sanity/base                0.142.1 (up to date)
@sanity/cli                 0.142.0 (up to date)
@sanity/components          0.142.1 (up to date)
@sanity/core                0.142.1 (up to date)
@sanity/default-layout      0.142.1 (up to date)
@sanity/default-login       0.142.1 (up to date)
@sanity/desk-tool           0.142.1 (up to date)
@sanity/production-preview  0.142.0 (up to date)
@sanity/storybook           0.142.0 (up to date)
@sanity/vision              0.142.1 (up to date)

**Comment by haakonmb** [comment]
Some additional versioning info:
*OS version*:
`sw_vers`:
```ProductName:	Mac OS X
ProductVersion:	10.13.6
BuildVersion:	17G2307```

*Node*:
`node -v`: `v8.11.4`

**Comment by rexxars** [comment]
Fixed in #1469
